src/main.js

```diff
@@ -118,7 +118,7 @@ function addToolbarButton() {
           {capture: true}
         )
 
-        toolbarGroup.append(clonedNode)
+        toolbarGroup.prepend(clonedNode)
```

Comment:
Many of us are used to having the GIF button where it is 😅 Ideally I think we'd look at how to add the GIF button to the `...` overflow menu.